1-exercises/B-object-arrays/exercise-2.js

```diff
@@ -16,10 +16,9 @@ let store = [
 
 let money = 1.5;
 
-let thingsICanBuy =
-  /* todo */
+let thingsICanBuy = store
+  .filter((canBuy) => canBuy.price <= money)
+  .map((canBuy) => canBuy.item);
 
-  // DO NOT EDIT BELOW THIS LINE
-  console.log(
-    `Expected output: Milk,Chocolate. Actual output: ${thingsICanBuy}`
-  );
+// DO NOT EDIT BELOW THIS LINE
+console.log(`Expected output: Milk,Chocolate. Actual output: ${thingsICanBuy}`);
```

Comment:
The improve code readability. You may consider rename variable **canBuy** to **item** on line 20 and **canBuy** to **itemCanBuy** on line 21